site/content/docs/include/http-healthcheck.en.md

```diff
@@ -1,15 +1,4 @@
-<div class="separator"></div>
-
-<a id="http" href="#http" class="field">`http`</a> <span class="type">Boolean or Map</span>  
-The http section contains parameters related to integrating your service with an Application Load Balancer. 
-
-To disable the Application Load Balancer, specify `http: false`. Note that for a Load-Balanced Web Service,
-at least one of Application Load Balancer or Network Load Balancer must be enabled.
-
-<span class="parent-field">http.</span><a id="http-path" href="#http-path" class="field">`path`</a> <span class="type">String</span>  
-Requests to this path will be forwarded to your service. Each Load Balanced Web Service should listen on a unique path.
-
-<span class="parent-field">http.</span><a id="http-healthcheck" href="#http-healthcheck" class="field">`healthcheck`</a> <span class="type">String or Map</span>  
+<span class="parent-field">http.</span><a id="http-healthcheck" href="#http-healthcheck" class="field">`healthcheck`</a> <span class="type">String or Map</span>
```

Comment:
Would you mind re-adding `healtcheck.port` here as well 😅 
https://github.com/aws/copilot-cli/pull/3564/files

we removed it for v1.18.1 but need to add it back